articles/conditional-automation-in-driving/index.md

```diff
@@ -0,0 +1,92 @@
+- **Level 4: High Automation**- All critical functions are automated. The driver does not control the car at any time.
+- **Level 5: Full automation**- This is the highest level of automation. All functions are automated, and the car can carry human passengers with no human interaction. 
+
+### Introduction to Conditional Automation in Driving
+Conditional automation is the third level of automation in driving. This level consists of approximately 75% automation. In conditional automation, the level 3 car performs safety-critical functions such as acceleration, deceleration, and steering. A level 3 car is also able to monitor the road and surroundings. The human driver is required to be aware in case of self-driving failures. When the car senses danger, the driver will be required to retake control.
+
+In level 3 cars, the human driver is simply a co-driver who will only take control under certain conditions. The driver can engage in other things like watching the surroundings or texting. The autonomous car will handle situations that require an immediate response. For example, when another vehicle suddenly appears in front of the car, the vehicle will perform emergency braking.
+
+### Key Features of a Level 3 Autonomous Car
+A level 3 autonomous car has certain technological features that facilitate self-driving. 
+
+#### Computer Power
+A computer system is required in a level 3 car to enhance the processing of information and activities within the car. Extreme computing power is required to support the automation of controls (more than the traditional CPUs).
+
+Graphical processing units (GPUs) are used by automobile companies for processing. However, GPUs have not been effective enough to provide the sufficient processing power needed in self-driving cars. This made Tesla to use neural network acceleration chips (NNA), which has a higher processing capacity than GPU. 
+
+#### Cameras
+Cameras are used to capture the road and surrounding environments. The captured images help the autonomous car to choose the right control depending on the situation. For example, if the camera senses an image in front of the car, a braking control will be initiated. The visual data collected using cameras is translated into 3D data that is actionable. 
+
+#### Radar
+The level 3 car uses radar to detect the distance and presence of objects that are in front of the car. Radar helps the car to achieve control in times of snow or rain. 
+
+#### Sensors
+Various sensors are used to detect situations and respond to them. For example, in case of failure, the car will use sensors to detect it, which will prompt the driver to take control. LIDAR sensors are placed at the top of the car to provide 3D data on the area surrounding the car. Inertial sensors, ultrasonic sensors, and GPS tracking sensors are utilized to monitor the performance of the autonomous car.

```

Comment:
what failure? specify the type of failure.

Reply:
I have modified the sentence.